protocol-units/execution/maptos/framework/releases/feature-flags/src/lib.rs

```diff
@@ -162,7 +162,7 @@ where
 		let now_u64 = std::time::SystemTime::now()
 			.duration_since(std::time::UNIX_EPOCH)
 			.map_err(|e| ReleaseBundleError::Build(e.into()))?
-			.as_secs();
+			.as_micros() as u64;
 		let expiration_timestamp_secs = now_u64 + expiration_timestamp_sec_offset;
```

Comment:
This value seems wrongly computed now, if `expiration_timestamp_sec_offset` is still in seconds.

We could benefit from a `Timestamp` domain type with controlled conversions.